scipy/stats/_morestats.py

```diff
@@ -2799,7 +2812,7 @@ def levene(*samples, center='median', proportiontocut=0.05):
 
     >>> fig, ax = plt.subplots(figsize=(8, 5))
     >>> lev_plot(ax)
-    >>> pvalue = dist.cdf(-res.statistic) + dist.sf(res.statistic)
```

Comment:
This is a one-sided "greater" test, so we only need the `sf`. In any case, `-res.statistic` would only make sense if the null distribution were symmetric.